pkgs/top-level/pkg-config/pkg-config-data.json

```diff
@@ -726,11 +726,6 @@
         "openssl"
       ]
     },
-    "orte": {
-      "attrPath": [
-        "openmpi"
```

Comment:
Continuing from @markuskowa's comment, maybe if you'd put `"prrte"` here instead of `"openmpi"` it will work?

Reply:
Our `prrte` does not seem to include that pkg config file either.

Reply:
Yes, `prrte` does not provide a pkg-config file.